docs/source/index.rst

```diff
@@ -0,0 +1,34 @@
+.. Hyperledger Indy Node documentation master file, created by
+   sphinx-quickstart on Wed Nov  7 15:20:42 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Hyperledger Indy Node's documentation!
+=================================================
+
+
+.. toctree::
+   :maxdepth: 1
+
```

Comment:
I think the following order would be more convenient:
    transactions.md
    requests.md
    auth_rules.md
    pool-upgrade.md
    start-nodes.md 
    add-node.md
    helper-scripts.md
    setup-iptables.md
    node-monitoring-tools-for-stewards.md
    node-state-diagnostic-tools.md
    ci-cd.md
    indy-file-structure-guideline.md
    write-code-guideline.md
    setup-dev.md
    1.3_to_1.4_migration_guide.md